### PR TITLE
Remove unused variable in RegQueryValueExA()

### PR DESCRIPTION
**Pull request by HenryJacques**
See #2826

### CODE REVIEW

**Comment by bmiklautz** [comment]
@freerdp-bot test.

**Comment by mfleisz** [comment]
+1

File: winpr/libwinpr/registry/registry.c [hunk 1]
```diff
@@ -266,13 +266,10 @@ LONG RegQueryValueExW(HKEY hKey, LPCWSTR lpValueName,
 LONG RegQueryValueExA(HKEY hKey, LPCSTR lpValueName,
 		LPDWORD lpReserved, LPDWORD lpType, LPBYTE lpData, LPDWORD lpcbData)
 {
-	Reg* reg;
 	RegKey* key;
 	RegVal* pValue;
 
 	key = (RegKey*) hKey;
-	reg = RegGetInstance();
```
Comment (akallabeth) [inline winpr/libwinpr/registry/registry.c]: This also removes the static initializer for the registry emulation.

Comment (HenryJacques) [inline winpr/libwinpr/registry/registry.c]: Isn't this done in RegOpenKeyExA()?

Comment (akallabeth) [inline winpr/libwinpr/registry/registry.c]: @HenryJacques Yes, you're right.
+1

**Pull request merged**